res/demo/app.json

```diff
@@ -3,14 +3,14 @@
     "startup_app": {
         "name": "Notifications Demo",
         "description": "Demo of Openfin Notifications",
-        "url": "http://localhost:9048/demo/app-launcher.html",
```

Comment:
Are we sure we want this to be 1337? That would prevent running local layouts and local notifs projects simultaneously.

Reply:
+1 on this